community/blog/2020-03-Releasing-Arcus-Messaging-and-BackgroundJobs-v.0.1.0/README.md

```diff
@@ -0,0 +1,89 @@
+# Announcing Arcus Messaging & Background Jobs
+
```

Comment:
How about this?

```suggestion

At Codit, we have been doing messaging since the company has started - Initially by using Microsoft BizTalk Server but over time we've been writing apps built on messaging in Azure as well.

We've seen that for a lot of customers we have to write the same boilerplate for processing message queues over and over again and it is not a trivial thing to do.

That's why we've decided to use our experience and start Arcus Messaging and Arcus Background Jobs which provides all of the boilerplate allowing you to focus on the application that you are building!

```